components/organisms/o-confirm-order.vue

```diff
@@ -260,13 +260,13 @@
     </div>
     <div class="actions">
       <paypal-button :styling="{ color: 'black' }" />
-      <SfButton
+      <!--SfButton
```

Comment:
please add a comment as to why this is commented out. also, if we can potentially do this : ` :disabled="$v.orderReview.$invalid || !productsInCart.length"` in the paypal button, that would be amazing.

Reply:
adding that doesn't work to disable the paypal button. looks like maybe we need to add an extra script somewhere? or add a validate in the paypal section? https://github.com/paypal/paypal-checkout-components/issues/350

Reply:
we can maybe add that as an issue to address later?